src/styles/main.scss

```
@@ -1,3 +1,76 @@
+$size: 100px;
+$gap: 1px;
+$calendar_padding: 10px;
+$calendar_size: $size * 7 + $calendar_padding * 2;
+
+@mixin square($size) {
+  height: $size;
+  width: $size;
+}
+
 body {
   margin: 0;
 }
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  width: $calendar_size;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+
+  $days: (
+    "tue": 0,
+    "wed": 1,
+    "thu": 2,
+    "fri": 3,
+    "sat": 4,
+    "sun": 5,
+    "mon": 6,
+  );
+
+  @each $day, $index in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: $size * $index + $gap * $index;
```

```suggestion
      margin-left: ($size + $gap) * $index;
```

margin-left should be 0 for monday, 101 for tuesday and so on

---

The task can make some missunderstanding becouse its saying Monday is the 1st, Friday is the 5th. At pic we see 1st at right side :P